util/rowDecoder/decoder.go

```diff
@@ -143,6 +138,22 @@ func (rd *RowDecoder) DecodeAndEvalRowWithMap(ctx sessionctx.Context, handle int
 	return row, nil
 }
 
+func (rd *RowDecoder) tryDecodeFromHandle(dCol Column, handle kv.Handle) bool {
+	if handle == nil {
+		return false
+	}
+	colInfo := dCol.Col.ColumnInfo
+	if dCol.Col.IsPKHandleColumn(rd.tbl.Meta()) {
+		if mysql.HasUnsignedFlag(colInfo.Flag) {
+			rd.mutRow.SetValue(colInfo.Offset, uint64(handle.IntValue()))
```

Comment:
There is another `SetValue` outside, maybe it's to easy to get confused when to do `set`.  Or just use the decoded value as the return value, and let the caller set or put these value outside?

Reply:
Then the code would have more indent.

The `continue` follows the pattern at line 99.